src/openrct2-ui/scripting/CustomMenu.h

```diff
@@ -46,7 +48,7 @@ namespace OpenRCT2::Scripting
     {
         std::shared_ptr<Plugin> Owner;
         std::string Id;
-        CURSOR_ID Cursor{};
+        CursorID Cursor{};
```

Comment:
This should be probably default initialize to Undefined and not the first value.